lms/djangoapps/certificates/api.py

```diff
@@ -79,8 +79,8 @@ def get_certificates_for_user(username):
     ]
 
 
-def generate_user_certificates(student, course_key, course=None, insecure=False, generation_mode='batch',
-                               title='None', forced_grade=None):
+def generate_user_certificates(student, course_key, course=None, designation='None', insecure=False,
```

Comment:
Since we're already forking these 2 lines, we could bite the bullet and format these lines more cleanly:

```python
def foo(
        student,
        # ....,
        forced_grade=None,
):
    pass